tasks/windows-install.yml

```diff
@@ -0,0 +1,71 @@
+---
+- name: make the user
+  win_user:
+    description: "Runs BuildKite jobs. See https://buildkite.com/docs/agent/v3/windows."
+    name: "buildkite-agent"
+    # TODO: Single static file with no isolation between runs is unsafe (but that's unlikely to occur)
+    # TODO: remove the password file afterwards
+    password: "{{ lookup('password', '/tmp/bk-agent-password length=32 chars=ascii_letters,digits,hexdigits,punctuation') }}"
+    password_never_expires: yes
+    user_cannot_change_password: yes
+
+- name: grant administrator privileges to user
+  win_group_membership:
+    name: Administrators
+    members:
+      - buildkite-agent
+    state: "{{ 'present' if buildkite_agent_windows_grant_admin else 'absent' }}"
+  notify:
+    - restart-windows-buildkite
+
+- name: make the user home directory
+  win_file:
+    state: directory
+    path: 'c:\users\buildkite-agent'
```

Comment:
I learned things I didn't know about YAML and quotes and backslashes... https://docs.ansible.com/ansible/latest/user_guide/windows_usage.html#path-formatting-for-windows